# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1336347446 -7200
#      Mon May 07 01:37:26 2012 +0200
# Branch gui
# Node ID 772ce0204b3f2cf232fb580f1a1ac0fe90cea0c9
# Parent  32e53ccdf0ca4f6998d30032185ecce2cbe218e2
Removed version number and "Octave GUI"-labelling.

* MainWindow: Removed Octave GUI version number.
* OctaveMainThread: Passing "octave" as first parameter to octave_main.

diff --git a/gui/src/MainWindow.cpp b/gui/src/MainWindow.cpp
--- a/gui/src/MainWindow.cpp
+++ b/gui/src/MainWindow.cpp
@@ -21,18 +21,16 @@
 #include <QSettings>
 #include <QDesktopServices>
 #include <QFileDialog>
 #include <QMessageBox>
 #include "MainWindow.h"
 #include "FileEditor.h"
 #include "SettingsDialog.h"
 
-#define VERSION_STRING "Octave GUI (0.8.8)"
-
 MainWindow::MainWindow (QWidget * parent):QMainWindow (parent)
 {
   // We have to set up all our windows, before we finally launch octave.
   construct ();
   OctaveLink::instance ()->launchOctave();
 }
 
 MainWindow::~MainWindow ()
@@ -305,17 +303,17 @@ MainWindow::construct ()
 
   connect (m_filesDockWidget, SIGNAL (openFile (QString)), this, SLOT (openExistingFile (QString)));
   connect (m_historyDockWidget, SIGNAL (information (QString)), this, SLOT (reportStatusMessage (QString)));
   connect (m_historyDockWidget, SIGNAL (commandDoubleClicked (QString)), this, SLOT (handleCommandDoubleClicked (QString)));
   connect (saveWorkspaceAction, SIGNAL (triggered ()), this, SLOT (handleSaveWorkspaceRequest ()));
   connect (loadWorkspaceAction, SIGNAL (triggered ()), this, SLOT (handleLoadWorkspaceRequest ()));
   connect (clearWorkspaceAction, SIGNAL (triggered ()), this, SLOT (handleClearWorkspaceRequest ()));
 
-  setWindowTitle (QString (VERSION_STRING));
+  setWindowTitle ("Octave");
 
   addDockWidget (Qt::LeftDockWidgetArea, m_workspaceView);
   addDockWidget (Qt::LeftDockWidgetArea, m_historyDockWidget);
   addDockWidget (Qt::RightDockWidgetArea, m_filesDockWidget);
   setStatusBar (m_statusBar);
 
   readSettings ();
   updateTerminalFont();
diff --git a/gui/src/backend/OctaveMainThread.cpp b/gui/src/backend/OctaveMainThread.cpp
--- a/gui/src/backend/OctaveMainThread.cpp
+++ b/gui/src/backend/OctaveMainThread.cpp
@@ -22,12 +22,12 @@ OctaveMainThread::OctaveMainThread (QObj
 {
 }
 
 void
 OctaveMainThread::run ()
 {
   setlocale(LC_ALL, "en_US.UTF-8");
   int argc = 1;
-  const char *argv[] = { "OctaveGUI" };
+  const char *argv[] = { "octave" };
   emit ready();
   octave_main (argc, (char **) argv, 0);
 }
